posts/how-to-commit-code-to-a-git-repository-with-github-desktop.md

```diff
@@ -0,0 +1,144 @@
+
+To add a local repository from your file system, open the **Current Repository** panel by clicking on it in the top toolbar and following these steps:
+
+- Click the **Add** button in the **Current Repository** panel.
+- Select **Add Existing Repository** in the dropdown menu.
+- Browse to the location of your repository in the file system.
+
+![The Add Local Repository dialog box lets you choose a local repository.](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-add-repository.png)
+
+Alternatively, you also have the option of cloning a remote repository hosted on GitHub or another source by selecting **Clone Repository** from the **Add** dropdown. This will also add it to the list of repositories.
+
+After following these steps, your repository’s name should now appear under **Current Repository** in the top toolbar.
+
+## Step 2 — Selecting a branch
+
+Next, ensure that the current branch is the one where you intend to make your commit. The current branch is displayed on the **Current Branch** tab. You can click on this tab to switch branches or to create a new branch.
+
+![The Current Branch tab displaying the `main` branch.](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-current-branch-tab.png)
+
+If you want to create a new branch before committing, use the **New Branch** button in the **Current Branch** panel. Creating a new branch can be useful if you’re working on a new feature and want to keep your changes independent of the main branch, or if you’re making some experimental changes.
+
+When thinking about creating and titling a new branch, consider having it describe what the new code will be doing. For example, if we’re planning to commit some changes to the `README.md` file in our repository, we could use the **New Branch** button to create a new branch and name it `update-readme`.
+
+![The **Current Branch** panel in GitHub Desktop](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-current-branch.png)
+
+Once we’ve selected a branch, its name will be displayed in the **Current Branch** tab.
+
+## Step 3 — Selecting changes to commit
+
+Before committing, you can choose which changed files you want to include in the commit.
+
+The **Changes** panel lists every changed file in your working directory. The header of the list displays a file count, like “3 changed files”, as displayed in the screenshot below.
+
+![The Changes tab showing the list of changed files.](https://storage.googleapis.com/sourcegraph-assets/learn/tutorial-images/github-desktop-changes.png)
+
+Different types of changes are represented by icons to the right of each file name.
+
+- Modified files are shown with a yellow dot in a square.
+- Untracked (newly created files) files are shown with a green `+` sign in a square.
```

Comment:
Plus and minus should be spelled out with the signs in parentheses or otherwise offset (with a dash / comma / colon)